nemo_skills/inference/prompt/few_shot_examples/examples_tabmwp.py

```diff
@@ -0,0 +1,217 @@
+$155 | 22,600 | 5,800
+$275 | 20,500 | 9,400
+$395 | 18,400 | 13,000
+$515 | 16,300 | 16,600
+$635 | 14,200 | 20,200
+""".strip(),
+        "reference_solution": "At the price of $155, the quantity demanded is greater than the quantity supplied. There is not enough of the good or service for sale at that price. So, there is a shortage.\n",
+        "generated_solution": """
+Let's solve this problem using Python code and the values from the table.
+{start_code}
+quantity_demanded_price_155 = 22600
+quantity_supplied_price_155 = 5800
+if quantity_demanded_price_155 > quantity_supplied_price_155:
+    result = 'shortage'
+else:
+    result = 'surplus'
+result
+{end_code}
+{start_code_output}
+surplus
+{end_code_output}
+So there is a \\boxed{surplus} at a price of 155 dollars.
+""".strip(),
+    },
+    {
+        "question": """Read the table regarding "Cans of food collected" and then write Python code to answer the following question:
+
+Question: Samir's class recorded how many cans of food each student collected for their canned food drive. What is the median of the numbers?""",
+        "table": """
+Samir | 7
+Kristen | 4
+Dakota | 7
+Jamie | 8
+Maggie | 9
+""".strip(),
+        "reference_solution": "Read the numbers from the table.\n\n7, 4, 7, 8, 9\n\nFirst, arrange the numbers from least to greatest:\n\n4, 7, 7, 8, 9\n\nNow find the number in the middle.\n\n4, 7, 7, 8, 9\n\nThe number in the middle is 7.\n\nThe median is 7.",
+        "generated_solution": """
+Let's solve this problem using Python code and the values from the table.
+{start_code}
+cans = [7, 4, 5, 8, 9]
+cans = sorted(cans)
+middle1 = (len(cans) - 1) // 2
+middle2 = len(cans) // 2
+result = (cans[middle1] + cans[middle2]) / 2
+result
+{end_code}
+{start_code_output}
+7
+{end_code_output}
+The median number of cans collected is \\boxed{7}.
+""".strip(),
+    },
+    {
+        "question": """Read the table regarding "" and then write Python code to answer the following question:
+
+Question: Lorenzo has $13.50. Does he have enough to buy a toy guitar and a set of juggling balls? Choose from the the options: ['yes', 'no']""",
+        "table": """
+toy boat | $5.54
+toy guitar | $8.23
+set of juggling balls | $5.01
+trivia game | $8.18
+jigsaw puzzle | $5.30
+toy dinosaur | $3.00
+""".strip(),
+        "reference_solution": "Add the price of a toy guitar and the price of a set of juggling balls:\n\n$8.23 + $5.01 = $13.24\n\n$13.24 is less than $13.50. Lorenzo does have enough money.",
+        "generated_solution": """
+Let's solve this problem using Python code and the values from the table.
+{start_code}
+guitar_price = 8.23
+juggling_balls = 5.01
+total_money = 13.5
+if total_money > juggling_balls + guitar_price:
+    result = 'yes'
+else:
+    result = 'no'
+result
+{end_code}
+{start_code_output}
+yes
+{end_code_output}
+\\boxed{Yes}, Lorenzo does have enough money to buy a toy guitar and juggling balls.
+""".strip(),
+    },
+]
+
+
+only_code = [
```

Comment:
I think this will not work in the current code version - have you used this few-shots? If not, maybe remove them

Reply:
Odd, it seemed to be working for me a few weeks back--I used it to make prompts for synthetic data generation. When I checked there was definitely a difference between the results from `only_code` vs. `text_with_code`.

What is expected behavior for the current version?

Reply:
the code you generate is not going to be run unless you wrap it with the special tokens. Since there are no such tokens here, the model will still learn to produce the code, but will never see its output

Reply:
Yeah, that checks out. I don't think we use this format so I'll remove these examples.